org.testeditor.web.dropwizard.xtext/src/main/java/org/testeditor/web/xtext/index/persistence/GradleProjectIndexUpdater.xtend

```diff
@@ -0,0 +1,78 @@
+package org.testeditor.web.xtext.index.persistence
+
+import com.google.common.io.CharStreams
+import java.io.File
+import java.io.InputStreamReader
+import java.nio.charset.StandardCharsets
+import java.nio.file.Files
+import java.nio.file.Paths
+import java.nio.file.StandardOpenOption
+import java.util.concurrent.TimeUnit
+import javax.inject.Inject
+import org.testeditor.web.xtext.index.CustomStandaloneBuilder
+import javax.inject.Singleton
+import org.slf4j.LoggerFactory
+
+@Singleton
+class GradleProjectIndexUpdater extends IndexUpdater {
+	
+	static val logger = LoggerFactory.getLogger(GradleProjectIndexUpdater)
+
+	@Inject CustomStandaloneBuilder builder
+	
+	static val GRADLE_PROCESS_TIMEOUT = 10 -> TimeUnit.MINUTES
```

Comment:
… or call it `GRADLE_PROCESS_TIMEOUT_MINUTES`, then you don't have to access these values using `.key` / `.value`, which seems a bit odd.

Reply:
thought this looked fancy :-)